site/topics/particle-swarm-optimization/pso.rst

```diff
@@ -27,6 +27,47 @@ Particles
 Velocity
 ========
 
+* The velocity determines where the particle will be for the next iteration of the algorithm
+* In other words, the velocity :math:`\vec{v}(t)` is used to determine the position of particle :math:`\vec{x}(t+1)`
+
+
+Velocity Calculation
+--------------------
+
+* Velocities are typically initialized with some random values within some range
+* But as the algorithm executes, the velocity of the particles change as they become influenced by
+
+    * The particles' best known position in space
+    * The population's best known position in space
```

Comment:
Emphasis on _particle_ and _population_ here would make this a little clearer. Currently it takes two reads.